packages/salesforcedx-vscode-apex/src/languageServer.ts

```diff
@@ -28,6 +29,7 @@ async function createServer(
 ): Promise<Executable> {
   try {
     deleteDbIfExists();
+    generateFauxClasses();
```

Comment:
Placeholder for now to be able to invoke the generation until we figure out where to put it.

Reply:
Please remove this line. We cannot have it accidentally try to generate the faux classes on start up now. That will be too slow. We will think of a different gating mechanism later.

Also, you cannot just call the generateFauxClasses here since it's possible that you don't even have a scratch org set up yet. So the call would fail.

Reply:
I have it as an async function so it's not blocking startup. Would you be ok with it behind an environment variable? Or do you want it gone completely until we figure out how to expose it?